dom-event-delegation/main.js

```diff
@@ -0,0 +1,8 @@
+var $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', function (event) {
+  console.log('event.target', event.target);
+  console.log('event.target.tagName', event.target.tagName);
```

Comment:
your solution is missing the conditional statement which checks if the correct element was clicked on. Make sure to add that functionality.